gulpfile.js

```diff
@@ -507,6 +513,46 @@ function release_deb(arch) {
     }));
 }
 
+function release_rpm(arch, done) {
+
+    // Check if dpkg-deb exists
+    if (!commandExistsSync('rpmbuild')) {
+        console.warn('rpmbuild command not found, not generating rpm package for ' + arch);
+        return done();
+    }
+
+    // The buildRpm does not generate the folder correctly, manually
+    createDirIfNotExists(RELEASE_DIR);
+
+    var options = {
+             name: pkg.name,
+             version: pkg.version,
+             buildArch: getLinuxPackageArch('rpm', arch),
+             vendor: pkg.author,
+             summary: pkg.description,
+             license: 'GNU General Public License v3.0',
+             requires: 'libgconf-2-4',
+             prefix: '/opt',
+             files:
+                 [ { cwd: path.join(APPS_DIR, pkg.name, arch),
+                     src: '*',
+                     dest: '/opt/betaflight/blackbox-log-viewer' } ],
+             postInstallScript: ['xdg-desktop-menu install /opt/betaflight/blackbox-log-viewer/blackbox-log-viewer.desktop'],
+             preUninstallScript: ['xdg-desktop-menu uninstall blackbox-log-viewer.desktop'],
+             tempDir: path.join(RELEASE_DIR,'tmp-rpm-build-' + arch),
+             keepTemp: false,
+             verbose: false,
+             rpmDest: RELEASE_DIR
+    };
+
+    buildRpm(options, function(err, rpm) {
```

Comment:
`buildRpm` needs to be defined: `const buildRpm = require('rpm-builder')`.

Also, turns out that `buildRpm` does not like the `-rc1` in the package version. We'll have to sanitise this for `buildRpm`, or stop using SemVer prereleases.

Reply:
Thanks.